src/qt/dataobj.cpp

```diff
@@ -58,32 +57,32 @@ wxDataFormat::wxDataFormat( wxDataFormatId formatId )
 
 wxDataFormat::wxDataFormat(const wxString &id)
 {
-    m_MimeType = wxQtConvertString(id);
+    m_MimeType = id;
 }
 
 wxDataFormat::wxDataFormat(const wxChar *id)
 {
-    m_MimeType = wxQtConvertString((wxString)id);
+    m_MimeType = (wxString)id;
```

Comment:
This is wrong, don't cast a `wxChar*` to `wxString`. Just completely remove the cast.